me.drewol.Unnamed-SDVX-Clone.yml

```diff
@@ -0,0 +1,46 @@
+id: me.drewol.Unnamed-SDVX-Clone
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: usc-game
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration
+  - --device=dri
+  # Needs to talk to the network
+  - --share=network
+  # Sound output
+  - --socket=pulseaudio
+modules:
+  - name: unnamed-sdvx-clone
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/zlib-ng/zlib-ng
+        tag: 2.1.3
+        dest: flatpak-deps/zlib-ng/
+      - type: archive
+        url: https://github.com/curl/curl/releases/download/curl-7_75_0/curl-7.75.0.tar.xz
+        sha256: fe0c49d8468249000bda75bcfdf9e30ff7e9a86d35f1a21f428d79c389d55675
+        dest: flatpak-deps/curl/
+      - type: git
+        url: https://github.com/Drewol/unnamed-sdvx-clone.git
+        commit: f264d380dd0550a781237028392e68db1ec51478
+      - type: file
+        path: me.drewol.Unnamed-SDVX-Clone.metainfo.xml
+      - type: archive
+        url: https://github.com/miloyip/rapidjson/archive/v1.1.0.tar.gz
+        sha256: bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e
+        dest: third_party/discord-rpc/thirdparty/rapidjson/
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DFETCHCONTENT_SOURCE_DIR_ZLIB=flatpak-deps/zlib-ng/
+      - -DFETCHCONTENT_SOURCE_DIR_CURL=flatpak-deps/curl/
+    post-install:
+      - install -D appimage/usc-game.png /app/share/icons/hicolor/128x128/apps/me.drewol.Unnamed-SDVX-Clone.png 
+      - install -D appimage/usc-game.desktop /app/share/applications/me.drewol.Unnamed-SDVX-Clone.desktop 
```

Comment:
you probably need to change the icon using the command `desktop-file-edit`.

Reply:
since I'm using `rename-icon` it seems like that's done for me, at least when I tested locally that's the case

Reply:
yes it does